### ISSUE TITLE
Create a class that hold the characters

### ISSUE THREAD

**Issue by cphyc** [y-js/y-richtext#31]
A new class should contain the characters with the following structure:

```
Characters = {
   characters = [Char]
}
Char = {
  char = String,
  left = [Selection],
  right = [Selection]
}
```

**Comment by cphyc** [comment]
See 32311e29502cd00d2eaae2041167ae75dfc56a78 and 3454e18a46f3c76c1acbcc6183f4e4eefd58f4d6.

**Comment by dmonad** [comment]
The approach I described in #32 is more minimalistic and doesn't require you to store the left/right of an selection. Furthermore, you should store a selection on a reference (because you want to traverse through the characters)

**Comment by cphyc** [comment]
What do you mean by a reference?

**Comment by cphyc** [comment]
Closed by pull request #35.
